source/_docs/settings-php.md

```diff
@@ -112,27 +112,29 @@ Yes, but only if at least one other file (e.g. `settings.php`) is present within
 
 #### How can I write logic based on the Pantheon server environment?
 
-Depending on your use case, there are two possibilities:
+Depending on your use case, there are three possibilities:
 
-For web only actions, like redirects, check for the existence of $\_SERVER['PANTHEON\_ENVIRONMENT']. If it exists, it will contain a string with the current environment (Dev, Test, or Live).
+ - For web only actions, like redirects, check for the existence of `$\_SERVER['PANTHEON\_ENVIRONMENT']`. If it exists, it will contain a string with the current environment (Dev, Test, or Live):
 
-    // Pantheon - web only.
-    if (isset($_SERVER['PANTHEON_ENVIRONMENT'])) {
-      // Only on dev web environment.
-      if ($_SERVER['PANTHEON_ENVIRONMENT'] == 'dev') {
-        // Custom code.
-      }
-    }
+        // Pantheon - web only.
+        if (isset($_SERVER['PANTHEON_ENVIRONMENT'])) {
+          // Only on dev web environment.
+          if ($_SERVER['PANTHEON_ENVIRONMENT'] == 'dev') {
+            // Custom code.
+          }
+        }

```

Comment:
Also note that `$_SERVER` is generally unavailable when executing code via the command line, use $_ENV wherever possible